languages/ruby/ruby_on_rails.md

```diff
@@ -3,34 +3,38 @@
 # Overview
 This document is a living doc shared by all developers at Lab Zero.  We as developers at Lab Zero believe that following these principles produces higher quality software.  It’s a pact that we share.  Please read and add to it as we find things on our projects that we’d like to make standard for our engineering practice.
 
-# Like Camping
+# Like Camping ⛺
 When camping there is a rule to leave your campsite cleaner than it was when you got there.  The same is true when writing code.  Each commit should be refactoring the minutia as you go such that each commit leaves the code base cleaner than when you started.
 
 # Documentation
-There are many things that we will need to communicate and share with others developers during and after we’re done.  Please document the basics that help folks know how to install, configure and deploy the app.  We are committed to updating these documents when things change to keep the docs useful.  Given that that’s a hard task, please use documentation wisely.  Too much and it’s unusable and all out of date.  If a dragon hides in the system, write about it.  There are three places that we’ll likely want to send folks: the app’s README, the repo’s Github wiki, and inline code comments. 
+There are many things that we will need to communicate and share with others developers during and after we’re done.  Please document the basics that help folks know how to install, configure and deploy the app.  We are committed to updating these documents when things change to keep the docs useful.  Given that that’s always going to be an ongoing task, please use documentation wisely.  Too much and it’s unusable and ends up out of date.  If a dragon hides in the system, write about it.  This starts with the repo's README.md but doesn't end there.  In some organizations a wiki may also be appropriate.  The integration tests especially can serve as a  
```

Comment:
Assume still working on the last sentence? 

Also, I know with documentation we were even more explicit with certain items and used a `doc` or `readme` folder? 

e.g. `/app_name/readmes`

Reply:
`For more complex functionality and/or code that might touch external functionality, it might be recommended to extend this approach by creating a separate set of README(s) as needed. Ideally, this would be referenced and linked from the base README.md`

Reply:
> Assume still working on the last sentence?
Thanks for reading so thoroughly! :-) I had moved part of that around and apparently lost some along the way.

How does that line up with wikis then?  Should we have all 3 of README.md, readmes/my_problem.md and wiki/my_problem ?

> Also, I know with documentation we were even more explicit with certain items and used a doc or readme folder?

I kinda think wiki and a docs directory should be either/or for code.  Though that doesn't mean a wiki might not be used for other institutional knowledge.
Between the two, I personally prefer the readmes/ directory myself because its easier to update while coding, which in turn means its slightly more likely to be up to date.

Reply:
@labzero/lz-engineering  If we have some level of consensus on a docs/ readmes/ directory I'm happy to add that as well.